branchpro/apps/_simulation.py

```diff
@@ -0,0 +1,121 @@
+#
+# SimulationApp
+#
+# This file is part of BRANCHPRO
+# (https://github.com/SABS-R3-Epidemiology/branchpro.git) which is released
+# under the BSD 3-clause license. See accompanying LICENSE.md for copyright
+# notice and full license details.
+#
+
+import pandas as pd
+import dash
+import dash_bootstrap_components as dbc
+
+import dash_core_components as dcc
+import dash_html_components as html
+
+import branchpro as bp
```

Comment:
Pep8 says this about importing:
```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.
```

I don't think we've followed this exactly in all our previous files, but it would be easy enough to adjust here